rj_gameplay/stp/rc.py

```diff
@@ -133,24 +233,90 @@ class GameRestart(Enum):
 class Field:
     """Information about the field."""
 
-    __slots__ = ["length_m"]
+    __slots__ = ["__length_m","__width_m","__goal_width_m"]
+
+    __length_m: float
+    __width_m: float
+    __goal_width_m: float
+
+    """
+    A list of the information avalible in the FieldDimensions.msg
+    we probably need many of these included in WorldState
+    float32 length
+    float32 width
+    float32 border - What is border?
+    float32 line_width - Does gameplay need this? 
+    float32 goal_width
+    float32 goal_depth - Does gameplay need this?
+    float32 goal_height - Does gameplay need this?
```

Comment:
I don't think that these things will be useful for now, if we need them it should be simple to add later.